modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/TeamMemberInputs/index.js

```diff
@@ -25,28 +22,7 @@ const TeamMemberInputs = ({
 	placeholderEmail,
 	selectOnChange,
 }) => {
-	const debouncedEmail = useDebounce(invite?.email, 500);
-	const [bannedDomain, setBannedDomain] = useState(debouncedEmail);
-
-	const [fetchBannedDomain, {data}] = useLazyQuery(getBannedEmailDomains);
-	const bannedDomainsItems = data?.c?.bannedEmailDomains?.items;
-
-	useEffect(() => {
-		const emailDomain = debouncedEmail.split('@')[1];
-
-		if (emailDomain) {
-			fetchBannedDomain({
-				variables: {
-					filter: `domain eq '${emailDomain}'`,
-				},
-			});
-
-			if (bannedDomainsItems?.length) {
-				setBannedDomain(bannedDomainsItems[0].domain);
-			}
-		}
-		// eslint-disable-next-line react-hooks/exhaustive-deps
-	}, [bannedDomainsItems, debouncedEmail]);
+	const bannedDomains = useBannedDomains(invite?.email, 500);
```

Comment:
I think it will be better if declare a constant for this value (500) , what do you think?

Reply:
Done!